github/actions_secrets.go

```diff
@@ -92,9 +92,9 @@ func (s *ActionsService) GetSecret(ctx context.Context, owner, repo, name string
 	return secret, resp, nil
 }
 
-// EncryptedSecret represents represents a secret that is encrypted using public key.
+// EncryptedSecret represents a secret that is encrypted using a public key.
 //
-// The value of EncryptedValue must value for your secret, encrypted with
+// The value of EncryptedValue must value of your secret, encrypted with
```

Comment:
I think this is still missing a word...

How about:  `The value of EncryptedValue must be your secret, ...`?

Reply:
I'm sorry, I need to learn to read :)

Reply:
Hey, I only know one language... you are doing awesome... thanks so much!!!